projects/astral-accessibility/src/lib/controls/screen-mask.component.ts

```diff
@@ -65,8 +66,16 @@ import { AstralCheckmarkSvgComponent } from '../util/astral-checksvg.component';
     `,
   imports: [NgIf, NgClass, AstralCheckmarkSvgComponent],
 })
-export class ScreenMaskComponent {
-  constructor(private renderer: Renderer2) {}
+export class ScreenMaskComponent extends AccessabilityComponent{
```

Comment:
Looking to merge in the prettier PR so that these styling fixes can be done automatically
```suggestion
export class ScreenMaskComponent extends AccessabilityComponent {
```